### PR TITLE
Feat navigation

### PR DESCRIPTION
**Pull request by Samarth-5109**
Screenshot (45)

### CODE REVIEW

**Review by jeetbhalani15 — APPROVED** [review]
good to go

File: styles/doc.css [hunk 1]
```diff
@@ -1,92 +1,116 @@
...
+
+.type-title {
+  font-size: 1.6rem;
+}
+
+.main-container {
+  margin: 1.5rem;
+}
+.types-container {
+  /* display: flex; */
```
Comment (tewarimohit) [inline styles/doc.css:111]: remove unnecessary comments....

**Review by tewarimohit — APPROVED** [review]
Just go through suggestion once...  change it... I'm approving...

File: styles/doc.css (hunk 1 above)
Comment (Samarth-5109) [reply styles/doc.css:111]: will do

**Pull request merged**